### DISCUSSION TITLE
Difficulty in taking the thigh, for example, in SPHinxsys2

### DISCUSSION THREAD

**Discussion by Mirzaei97** [Xiangyu-Hu/SPHinXsys#444] [Ideas]
Dear Xiangyu

I hope all is fine. Thank you for your reply. It seems that the .vtp file is needed to display results in Paraview. In SPHinxsys, example 15 (aortic-valve 2D) there isn't a .vtp file.

Regards
Mirzaei

**Comment by Xiangyu-Hu** [comment]
It seems that the test is not on, You can can go to the bin folder to run the case directly.

**Comment by Xiangyu-Hu** [comment]
The test is included in newest update.

**Reply by Mirzaei97** [reply]
Thank you for your reply.
I'll check it and back again.

**Comment by Mirzaei97** [comment]
Dear Xiangyu
I hope everything is all right
I downloaded SPHinxsys from this link. Do you have another download link?
"https://github.com/Xiangyu-Hu/SPHinXsys.git sphinxsys"

**Reply by Xiangyu-Hu** [reply]
This is the only one.
